.github/pull_request_template.md

```diff
@@ -1,24 +1,39 @@
-Describe what the changes are meant to address. If the implementation is non trivial, please also include a short overview.
+**Select one or more of:**
 
+<!-- To check an item, fill the brackets with the letter x; the result should look like `[x]`. -->
 
+- [ ] Issue
+- [ ] Feature
+- [ ] Fix
+- [ ] Housekeeping
+- [ ] Other:
```

Comment:
Can this be done with a label instead?

Reply:
How would that look? Asking cuz im actually facing this problem at work